review.py

```diff
@@ -1,133 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Why this was removed?

Reply:
The script would create statics for Phabricator reviews E.g. how long are reviews open or how long would it take to get a first respond.